### DISCUSSION TITLE
Any update ?

### DISCUSSION THREAD

**Discussion by MrAhmedSayedAli** [silverqx/TinyORM#5] [General]
https://github.com/silverqx/TinyORM/blob/55d21463531d89111ceefbab2ef2998e716c6ad2/src/orm/connectors/connectionfactory.cpp#L42

**Comment by silverqx** [comment]
Hi, still no, why are you waiting so much for SqlServer support?

**Reply by MrAhmedSayedAli** [reply]
Most of my projects depend on it

**Comment by silverqx** [comment]
I have never used the SqlServer but I know its an amazing DB, add support for it will take ~6 weeks of every day 10 hours of work (of course weekend free and some sick days). I don't know when this will happen. It has a very low priority for me sry. 🙃
